services/node_manager/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+#ignite configuration
+ignite.host=localhost
+ignite.port=10800
+ignite.key-store-path=keystore.jks
+ignite.key-store-password=123456
+ignite.trust-store-path=truststore.jks
+ignite.trust-store-password=123456
+## MULTIPART (MultipartProperties)
+# Enable multipart uploads
+spring.servlet.multipart.enabled=true
```

Comment:
Do you need to use these new configurations like servlet.multipart.enabled?

Reply:
The following configurations are removed:
servelet.multipart.enable 
spring.servlet.multipart.file-size-threshold=2KB
spring.servlet.multipart.max-request-size=215MB